orbs/helm/circle-wait-job

```diff
@@ -65,7 +65,7 @@ class CLI
     end
 
     parser.on_tail("--version", "print version and exit") do
-      puts("v0.0.1")
+      puts("v0.0.2")
```

Comment:
**info**

The current version of this orb is actually the same as all other orbs and we're around version 0.20.x.

Reply:
I think this is the version of the script, not the orb. Don't you think?

Reply:
I don't know 😄  if you say that it's the script version, it's ok for me. It's purely informative ?

Reply:
Yes. I think it is just an information.